installer/install.sh

```
@@ -0,0 +1,635 @@
+    sudo rm -rf /var/www/mythicaldash-v3
+
+    # Remove cron jobs
+    (crontab -l | grep -v -e '/var/www/mythicaldash-v3/') | crontab -
+    
+    echo "Without-Docker based uninstallation complete."
+}
+
+uninstall_cloudflare_tunnel() {
+    echo "Uninstalling Cloudflare Tunnel..."
+    if [ -f /var/www/mythicaldash-v3/.cf_creds ]; then
+        . /var/www/mythicaldash-v3/.cf_creds
```

Shellcheck (0.11.0)

[info] Not following: /var/www/mythicaldash-v3/.cf_creds was not specified as input (see shellcheck -x).

(SC1091)